sdk/deviceupdate/Azure.IoT.DeviceUpdate/CHANGELOG.md

```diff
@@ -1,14 +1,14 @@
 # Release History
 
-## 1.0.0-beta.5 (Unreleased)
+## 1.0.0 (Unreleased)
```

Comment:
You might as well put the date in there now so you can save yourself another PR.

Reply:
What date should I put here? The official release date (2022-10-01), or today's date (since I want to merge and publish the packages asap)?

Reply:
Ok, I guess I will use the date when we want to release (publish the library), which is tomorrow.